benches/spresso_benchmark.rs

```diff
@@ -1,42 +1,42 @@
-use criterion::{criterion_group, criterion_main, Criterion, BenchmarkId};
```

Comment:
Would be better to comment out the whole file for now, instead of just the function bodies